dist/sizzle.js

```diff
@@ -482,16 +482,19 @@ function createDisabledPseudo( disabled ) {
 			//   https://html.spec.whatwg.org/multipage/forms.html#concept-option-disabled
 			// All such elements have a "form" property.
 			if ( elem.parentNode && elem.disabled === false ) {
-				return "label" in elem ?
 
-					// Option elements defer to a parent optgroup if present
-					( "label" in elem.parentNode ?
-						elem.parentNode.disabled === disabled :
-						elem.disabled === disabled ) :
+				// Option elements defer to a parent optgroup if present
+				if ( "label" in elem ) {
+					if ( "label" in elem.parentNode ) {
+						return elem.parentNode.disabled === disabled;
+					} else {
```

Comment:
Yeah, had the same question, I don't think it is established practise to remove redundant `else` statements, but it is not contradicts our style guide